test/integration/roles/test_openssl/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+- name: Generate a private key
```

Comment:
Format is `test/integration/targets/$module_name/
Move to `targets` i.e. 
`test/integration/targets/openssl/tasks/main.yml